src/cbl-mariner/manifest.json

```diff
@@ -394,6 +394,32 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/cbl-mariner/2.0/cross/freebsd/13/amd64",
+              "os": "linux",
+              "osVersion": "cbl-mariner2.0",
+              "tags": {
+                "cbl-mariner-2.0-cross-amd64-freebsd-13-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {},
```

Comment:
There's still usage of the old `ubuntu-18.04...`  tags within various repos. Will you be updating those as well?

Reply:
That's the idea. 🙂